File: Python/7py 3/perimeter_sequence.py
"""
|--------------------------------------------------------------------------
| Perimeter sequence
|--------------------------------------------------------------------------
|
"""

def perimeter_sequence(a, n): 
   return a * n * 4


perimeter_sequence = lambda a, n : 4 * a * n


def perimeterSequence(a, n):
   res = 0
   for i in range(n):
      res += 4 * a
   return res


def perimeter_sequence(a, n): 
   return a * n << 2


print(perimeter_sequence(1, 3), 12)

File: JavaScript/7js 3/perimeterSequence.js
/*
|--------------------------------------------------------------------------
| Perimeter sequence
|--------------------------------------------------------------------------
|
*/

function perimeterSequence(a, n) {
   return a * n * 4;
}


const perimeterSequence = (a, n) => 4 * a * n;


function perimeterSequence(a, n) {
   let res = 0;
   for (let i = 0; i < n; i++) {
      res += 4 * a;
   }
   return res;
}


console.log(perimeterSequence(1, 3), 12);